### DISCUSSION TITLE
Project scaffolder

### DISCUSSION THREAD

**Discussion by hashrock** [jurassiscripts/velociraptor#85] [Ideas]
How about adding a function like `yarn create` or `npm init`?

> $ vr create oak-app

I think it would be useful to be able to easily generate a project that comes with `scripts.yaml` and a framework.

A separate CLI would probably suffice, but it might be smoother if you could just setup vr and start developing a deno project.

**Comment by umbopepato** [comment]
Hi @hashrock, thank you for the suggestion, that would be a really cool feature!
Like [CLI installation helpers](https://github.com/jurassiscripts/velociraptor/issues/63), we could use a dedicated script name (`init`?) in script configs. For example

```shell
$ vr init oak
```

would search for a vr scripts configuration in `https://<registry>/oak` and execute the `init` script, if present.
What do you say?

**Reply by hashrock** [reply]
Thanks! I think that would work just fine.
